src/index.ts

```diff
@@ -15,7 +15,7 @@ enum Message {
   INITIAL_CONNECTION_FAILED = 'initial-connection-failed'
 }
 
-interface WebRTCPlayerOptions {
+export interface WebRTCPlayerOptions {
```

Comment:
I am not a fan of passing through the WebRTCPlayerOptions interface directly down to the adapters. I would suggest to add a new type of option interface for example:

```
export interface MediaConstraints {
  audioOnly: boolean;
  videoOnly: boolean;
}
```
and then in the WebRTCPlayerOptions interface
```
interface WebRTCPlayerOptions {
...
  mediaConstraints?: MediaConstraints;
}
```
and then add to the AdapterFactoryFunction:
```
...
onError: (error: string) => void;
mediaConstraints: MediaConstraints;
```